### PR TITLE
fixes webhook execution issue

### PR DESCRIPTION
**Pull request by mitresh95**
**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Some changes resulted in events not actually getting queued. This fixes that.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Setup a webhook for when a contact is updated
2. Update a contact
3. Webhook is never delivered

#### Steps to test this PR:
1. Repeat and webhook is delivered

### CODE REVIEW

**Comment by mitresh95** [comment]
Thank you @npracht for correctly stating the title.

File: app/bundles/WebhookBundle/Model/WebhookModel.php [hunk 1]
```diff
@@ -246,9 +246,9 @@ public function queueWebhooks($webhookEvents, $payload, array $serializationGrou
 
             $webhook->addQueue($this->queueWebhook($webhook, $event, $payload, $serializationGroups));
 
-            // add the queuelist and save everything if command process
-            if ($this->queueMode == self::COMMAND_PROCESS) {
-                $this->saveEntity($webhook);
+            if (self::COMMAND_PROCESS === $this->queueMode) {
+                // Queue to the database to process later
+                $this->getQueueRepository()->saveEntity($this->queuedPayloads[$webhook->getId()]);
```
Comment (escopecz) [inline app/bundles/WebhookBundle/Model/WebhookModel.php]: As PHPSTAN check pointed out, where is the `queuedPayloads` property defined?

**Comment by escopecz** [comment]
@mitresh95 can you resolve this as we talked about it? The commit belongs to another PR, right?

**Comment by mitresh95** [comment]
@escopecz Yes.

**Comment by escopecz** [comment]
The commit moved from this PR to https://github.com/mautic/mautic/pull/6594

**Pull request closed without merging**